.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     parallelism: 1
     docker:
       - image: 'cypress/included:3.4.0'
-    working_directory: ~/wheel
+    working_directory: ~/neetoui-challenge
```

Comment:
This is wrong! The PR is titled `Create note deletion page UI` and a lot of these unrelated changes have been made as part of this PR itself.

PRs should be atomic. Moving forward, please don't follow this pattern! PRs should be made with care.